dGame/Entity.cpp

```diff
@@ -146,17 +146,15 @@ Entity::~Entity() {
 			return;
 		}
 
-		Entity* zoneControl = Game::entityManager->GetZoneControlEntity();
-		for (CppScripts::Script* script : CppScripts::GetEntityScripts(zoneControl)) {
-			script->OnPlayerExit(zoneControl, this);
+		auto* zoneControl = Game::entityManager->GetZoneControlEntity();
```

Comment:
Why the change to auto* here? Not an issue, just curious.

Reply:
idk, just felt like it